linux/system/cpu_frequency.sls

```diff
@@ -0,0 +1,10 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.cpu_frequency is defined %}
+
+ondemand_service_disable:
```

Comment:
I have tested it on both trusty and xenial. 4 xenial bare metal nodes. Verified if its working with command  cat /proc/cpuinfo | grep MHz 
Its working find on all 4 xenial nodes.